receiver/awscontainerinsightreceiver/README.md

```diff
@@ -352,6 +352,7 @@ kubectl apply -f config.yaml
 | rest_client_request_duration_seconds                      | Seconds |
 | rest_client_requests_total                                | Count   |
 | etcd_request_duration_seconds                             | Seconds |
+| apiserver_flowcontrol_rejected_requests_total             | Count   |
```

Comment:
Mind moving this up to be with the other apiserver metrics?

Reply:
done